### DISCUSSION TITLE
arduino upload to WT32-SC01-PLUS FAILS

### DISCUSSION THREAD

**Discussion by heintoussaint** [sukesh-ak/ESP32-TUX#21] [Q&A] [answered]
compilling is OK but upload fault comport fault or wt32 not connected
File "esptool\loader.py", line 351, in command

  File "serial\serialutil.py", line 372, in timeout

  File "serial\serialwin32.py", line 222, in _reconfigure_port

serial.serialutil.SerialException: Cannot configure port, something went wrong. Original message: OSError(22, 'Er is een niet-bestaand apparaat opgegeven.', None, 433)

[6820] Failed to execute script 'esptool' due to unhandled exception!
here the output from arduino IDE  (sorry in Dutch !)
other ESP32 v2 and others work fine
what do I WRONG ????
please help

de geselecteerde seriële poort [6820] Failed to execute script 'esptool' due to unhandled exception!

 bestaat niet of uw board is niet aangesloten.

**Comment by sukesh-ak** [comment]
Install the already built firmware from here to test. 

https://tux.sukesh.me/

Project is using ESP-IDF and not Arduino. Though the error is not related to the project or code.

**Comment by sukesh-ak** [comment, marked as answer]
You can also check this https://github.com/sukesh-ak/ESP32-TUX/issues/18
